insurance/pipeline.ipynb

```diff
@@ -77,24 +77,24 @@
     "    name='dkube-insurance-pl',\n",
     "    description='sample insurance pipeline with featuresets'\n",
     ")\n",
-    "def insurance_pipeline():\n",
+    "def insurance_pipeline(token):\n",
```

Comment:
can token be made optional ? token="" or token = None ? Would that work in both versions ? User needs to pass token with older version of DKube.